cli/core/checkpoint.go

```diff
@@ -48,26 +52,32 @@ func SubmitCheckpointProof(ctx context.Context, owner, eigenpodAddress string, c
 			bind.WaitMined(ctx, eth, txn)
 		}
 		tracing.OnEndSection()
-		color.Green("OK")
+		if verbose {
+			color.Green("OK")
+		}
 	}
 
-	if !noSend {
-		color.Green("Complete! re-run with `status` to see the updated Eigenpod state.")
-	} else {
-		color.Yellow("Submit these proofs to network and re-run with `status` to see the updated Eigenpod state.")
+	if verbose {
+		if !noSend {
+			color.Green("Complete! re-run with `status` to see the updated Eigenpod state.")
```

Comment:
maybe we should do this if it's verbose or not?

Reply:
nope

a good chunk of the people using the CLI are just throwing it in a CI / automated process

they are likely parsing out this and ignoring it anyways

anyone using the CLI manually (e.g for tests) likely should be using --verbose.